### ISSUE TITLE
Scrum Report #3

### ISSUE THREAD

**Issue by kenhyj** [kenhyj/Traceify#28]
Don't forget your csid!
1. What did you work on this past iteration (2 weeks)?
2. What were any major issues/challenges you ran into?
3. What do you plan to work on for this coming iteration (2 weeks)?

**Comment by kenhyj** [comment]
csid: e5c0b
1. I was able to get the functional part of the symptom checker done. 
2. eslint kept breaking the react app, I thought I fixed it. Upon merge it caused problems still. 
3. Focus on importing any real (csv or json) data or API from BC's government/health organization rather than mock data we make for ourselves.

**Comment by kessris** [comment]
CS id: k0b0b

1. Setting up OAuth, showing another menu bar option after logging in, using Google map API for autofill search, creating/populating db with mock data
2. eslint set-up
3. Connecting frontend, backend, db. Focus on styling.

**Comment by mandywan** [comment]
1. I created a custom Map marker component with  an info window, heat map layer, chips to control the view of the map, set up the redux for the map component, and some mock data. 

2. The npm package I used was not updated and did not really support heat map layer. Because of that, the entire maps would not load. I had to create a HOC to wrap around the map and render the maps in a completely different way than I did for the last sprint.

3. Finish styling for the entire maps, connect database to maps, create more custom markers, finish up the side bar maps, display KML layer or create my own polygons, and redesign the entire app’s UI.

**Comment by ChaoyuXiang** [comment]
csid : l0y2b
1. I changed thank you page (previously landing page) to modal page that appears only once when user open the website. I helped ken style symptom checker and added two actions to connect checker component and disclaimer result component. 
2. I found most different part was to handle multiple actions within a component class. 
3. In next sprint, i'll focus on Have I been exposed ? page and keep on styling and responsive design.
